setup_pulp.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
Question from me: Is is still common to have the interpreter under `/bin`? Have my bash under `/usr/bin` since I remember :) (`/bin` links to `/usr/bin`)

Reply:
I dunno. Less typing this way? I can modify it.